### PR TITLE
Guarding from crash if pod lib create has a + character in the name

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

**Comment by segiddins** [comment]
@fulldecent can you please add a test, a changelog entry, and give the commit a message that describes the content of the change rather than the specific bug it's fixing? Thanks!

**Comment by Coeur** [comment]
Suggested description:

> Guarding from crash if `pod lib create` has a `+` character in the name.

**Comment by fulldecent** [comment]
How is this?

**Comment by orta** [comment]
Looks great 👍

**Comment by segiddins** [comment]
👍🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
-* None.  
+* Guarding from crash if pod lib create has a + character in the name.
```
Comment (segiddins) [inline CHANGELOG.md]: missing trailing two spaces

**Comment by fulldecent** [comment]
@segiddins space added

**Comment by segiddins** [comment]
@fulldecent I don't know why travis isn't running, could you try rebasing it once more? Sorry

**Comment by fulldecent** [comment]
@segiddins Rebased for the refactoring in d29a8ae7e479115b38734721930925b6557abbe2

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Avoid removing all files when root directory contains unicode characters.  
   [Marc Boquet](https://github.com/marcboquet)
   [#5294](https://github.com/CocoaPods/CocoaPods/issues/5294)
+* Guarding from crash if pod lib create has a + character in the name.
```
Comment (segiddins) [inline CHANGELOG.md]: spacing here is off

**Comment by fulldecent** [comment]
Spacing corrected

**Comment by fulldecent** [comment]
Travis now passing. Good to go.

**Comment by orta** [comment]
Thanks!

**Pull request merged**